script.js

```diff
@@ -0,0 +1,97 @@
+const API_KEY = 'b6ad49d4';
+const RESULTS_PER_PAGE = 10;
+
+let currentSearchValue = '';
+let currentPage = 1;
+let currentResults = [];
+
+const searchInputEl = document.getElementById('search-input');
+const inputBtnEl = document.getElementById('input-button');
+const movieListEl = document.getElementById('movie-list');
+const paginationEl = document.getElementById('pagination');
+const detailsEl = document.getElementById('details');
+const containerEl = document.getElementsByClassName('container')[0];
+
+function renderError(data) {
+  movieListEl.innerHTML = '';
+  paginationEl.innerHTML = '';
+
+  movieListEl.innerText = data.Error;
+}
+
+function renderResults(data) {
+  movieListEl.innerHTML = '';
+  detailsEl.innerHTML = '';
+
+  data.Search.forEach((movie) => {
+    const itemEl = document.createElement('li');
+    const itemTitleEl = document.createElement('p');
+    const itemMoreBtnEl = document.createElement('a');
+
+    itemTitleEl.innerText = movie.Title;
+    itemMoreBtnEl.innerText = 'More';
+
+    itemMoreBtnEl.addEventListener('click', () => {
```

Comment:
Use event delegation technique here since you are creating a looooooooot of listeners in browser API